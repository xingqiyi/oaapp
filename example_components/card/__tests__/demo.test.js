import { webDemoTest } from '../../../tests/shared/demoTest';

webDemoTest('card');
