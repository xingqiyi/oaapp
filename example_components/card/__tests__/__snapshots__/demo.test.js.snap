// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/card/demo/basic.md correctly 1`] = `
<div
  class="am-wingblank am-wingblank-lg"
>
  <div
    class="am-whitespace am-whitespace-lg"
  />
  <div
    class="am-card"
  >
    <div
      class="am-card-header"
    >
      <div
        class="am-card-header-content"
      >
        <img
          src="https://cloud.githubusercontent.com/assets/1698185/18039916/f025c090-6dd9-11e6-9d86-a4d48a1bf049.png"
        />
        This is title
      </div>
      <div
        class="am-card-header-extra"
      >
        <span>
          this is extra
        </span>
      </div>
    </div>
    <div
      class="am-card-body"
    >
      <div>
        This is content of \`Card\`
      </div>
    </div>
    <div
      class="am-card-footer"
    >
      <div
        class="am-card-footer-content"
      >
        footer content
      </div>
      <div
        class="am-card-footer-extra"
      >
        <div>
          extra footer content
        </div>
      </div>
    </div>
  </div>
  <div
    class="am-whitespace am-whitespace-lg"
  />
</div>
`;

exports[`renders ./components/card/demo/full.md correctly 1`] = `
<div>
  <div
    class="am-whitespace am-whitespace-lg"
  />
  <div
    class="am-card am-card-full"
  >
    <div
      class="am-card-header"
    >
      <div
        class="am-card-header-content"
      >
        <img
          src="https://cloud.githubusercontent.com/assets/1698185/18039916/f025c090-6dd9-11e6-9d86-a4d48a1bf049.png"
        />
        This is title
      </div>
      <div
        class="am-card-header-extra"
      >
        <span>
          this is extra
        </span>
      </div>
    </div>
    <div
      class="am-card-body"
    >
      <div>
        This is content of \`Card\`
      </div>
    </div>
    <div
      class="am-card-footer"
    >
      <div
        class="am-card-footer-content"
      >
        footer content
      </div>
      <div
        class="am-card-footer-extra"
      >
        <div>
          extra footer content
        </div>
      </div>
    </div>
  </div>
</div>
`;
